web/src/flow/script.get-account-items.js

```diff
@@ -1,19 +1,7 @@
 import * as fcl from "@onflow/fcl"
 import {Address} from "@onflow/types"
 import {expandAccountItemsKey} from "src/hooks/useAccountItems"
-
-const FETCH_ACCOUNT_ITEMS_SCRIPT = `
-  import NonFungibleToken from 0xNonFungibleToken
-  import KittyItems from 0xKittyItems
-
-  pub fun main(address: Address): [UInt64] {
-    if let collection = getAccount(address).getCapability<&KittyItems.Collection{NonFungibleToken.CollectionPublic, KittyItems.KittyItemsCollectionPublic}>(KittyItems.CollectionPublicPath).borrow() {
-      return collection.getIDs()
-    }
-
-    return []
-  }
-`
+import FETCH_ACCOUNT_ITEMS_SCRIPT from "cadence/scripts/get_account_items.cdc"
```

Comment:
Ah, I guess `fcl.cdc` isn't even needed.

I think it'd be good to remove the calls to `fcl.cdc` from all of these files to keep things simple and consistent.

Reply:
will do